libs/h3/src/main/java/org/elasticsearch/h3/LatLng.java

```diff
@@ -74,6 +79,50 @@ public double getLonDeg() {
         );
     }
 
+    /**
+     * Computes the point on the sphere with a specified azimuth and distance from
```

Comment:
It would be nice with a reference to where this algorithm comes from

Reply:
It comes from the original H3 library, added a comment.